writeups/quic-bit-grease.txt

```diff
@@ -0,0 +1,175 @@
+summarize any WG discussion and conclusion regarding the IPR disclosures.
+
+There are no IPR disclosures for this document.
+
+(9) How solid is the WG consensus behind this document? Does it represent the
+strong concurrence of a few individuals, with others being silent, or does the
+WG as a whole understand and agree with it?
+
+The extension defined in this document has been implemented and deployed by a
+wide range of vendors. There is strong consensus on the document, with no
+dissent raised during the process.
+
+(10) Has anyone threatened an appeal or otherwise indicated extreme discontent?
+If so, please summarise the areas of conflict in separate email messages to the
+Responsible Area Director. (It should be in a separate email because this
+questionnaire is publicly available.)
+
+No.
+
+(11) Identify any ID nits the Document Shepherd has found in this document. (See
+http://www.ietf.org/tools/idnits/ and the Internet-Drafts Checklist).
+Boilerplate checks are not enough; this check needs to be thorough.
+
+No nits.
+
+(12) Describe how the document meets any required formal review criteria, such
+as the MIB Doctor, YANG Doctor, media type, and URI type reviews.
+
+N/A
+
+(13) Have all references within this document been identified as either
+normative or informative?
+
+Yes.
+
+(14) Are there normative references to documents that are not ready for
+advancement or are otherwise in an unclear state? If such normative references
+exist, what is the plan for their completion?
+
+No.
+
+(15) Are there downward normative references references (see RFC 3967)? If so,
+list these downward references to support the Area Director in the Last Call
+procedure.
+
+No.
+
+(16) Will publication of this document change the status of any existing RFCs?
+Are those RFCs listed on the title page header, listed in the abstract, and
+discussed in the introduction? If the RFCs are not listed in the Abstract and
+Introduction, explain why, and point to the part of the document where the
+relationship of this document to the other RFCs is discussed. If this
+information is not in the document, explain why the WG considers it unnecessary.
+
+No.
+
+(17) Describe the Document Shepherd's review of the IANA considerations section,
+especially with regard to its consistency with the body of the document. Confirm
+that all protocol extensions that the document makes are associated with the
+appropriate reservations in IANA registries. Confirm that any referenced IANA
+registries have been clearly identified. Confirm that newly created IANA
+registries include a detailed specification of the initial contents for the
+registry, that allocations procedures for future registrations are defined, and
+a reasonable name for the new registry has been suggested (see RFC 8126).
+
+This document registers a new Transport Parameter. This is a permanent
+registration in the range 0x00-0x3f, which requires Standards Action or IESG
+approval.
```

Comment:
Copy-paste from datagrams draft perhaps?

0x2ab2 is the value, which only requires a specification.

Reply:
It's a copy paste, combined with tired eyes scanning as far as "0x2" and figuring that was in range. Thanks for spotting it, I'll fix this up tomorrow